doc/release-notes/5.8-release-notes.md

```diff
@@ -0,0 +1,120 @@
+
+If you are running Payara as a non-root user (and you should be!), **remember not to execute the commands below as root**. Use `sudo` to change to that user first. For example, `sudo -i -u dataverse` if `dataverse` is your dedicated application user.  
+
+In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.
+
+`export PAYARA=/usr/local/payara5`
+
+(or `setenv PAYARA /usr/local/payara5` if you are using a `csh`-like shell)
+
+1\. Undeploy the previous version.
+
+- `$PAYARA/bin/asadmin list-applications`
+- `$PAYARA/bin/asadmin undeploy dataverse<-version>`
+
+2\. Stop Payara and remove the generated directory
+
+- `service payara stop`
+- `rm -rf $PAYARA/glassfish/domains/domain1/generated`
+
+3\. Start Payara
+
+- `service payara start`
+  
+4\. Deploy this version.
+
+- `$PAYARA/bin/asadmin deploy dataverse-5.8.war`
+
+5\. Restart payara
+
+- `service payara stop`
+- `service payara start`
+
+## Additional Release Steps
+
+1\. Replace Solr schema.xml. See specific instructions below for those installations without custom metadata blocks (1a) and those with (1b).
+
+1a\.
+
+For installations without Custom Metadata Blocks:
+
+- stop Solr instance (usually service solr stop, depending on Solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/5.8/installation/prerequisites.html#solr-init-script)
+
+- replace schema.xml with the file included in this release
+
+cp /tmp/dvinstall/schema.xml /usr/local/solr/solr-8.8.1/server/solr/collection1/conf
+
+- start Solr instance (usually service solr start, depending on solr/OS)
+
+2\. Run a full reindex, following the steps in the [Admin Guide](http://guides.dataverse.org/en/5.8/admin/solr-search-index.html).
+
+1b\.
+
+For installations with Custom Metadata Blocks:
+
+- stop Solr instance (usually service solr stop, depending on Solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/5.8/installation/prerequisites.html#solr-init-script)
+
+- replace schema.xml
+
+To include your custom metadata fields after updating schema.xml, you can use a simple `curl` command. Please download the `update-fields.sh` script before starting, or use it from the extracted installer.
+
+```
+curl "http://localhost:8080/api/admin/index/solr/schema" | update-fields.sh conf/schema.xml
+```
+
+Please adapt the above to point to your Dataverse installation and to the correct `schema.xml` file in your Solr installation. See the [Installation Guide](https://guides.dataverse.org/en/5.8/installation/prerequisites.html#installing-solr) for some hints about usual places or use `find / -name schema.xml`)
+
+cp /tmp/dvinstall/schema.xml /usr/local/solr/solr-8.8.1/server/solr/collection1/conf
```

Comment:
and this line isn't needed.

the paragraph above is duplicated.... ok to leave as is. or could be dropped, if we rearrange the first instance to cover both examples.